gnuradio-runtime/lib/vmcircbuf_mmap_shm_open.cc

```diff
@@ -34,15 +34,19 @@ namespace gr {
 vmcircbuf_mmap_shm_open::vmcircbuf_mmap_shm_open(int size) : gr::vmcircbuf(size)
 {
 #if !defined(HAVE_MMAP) || !defined(HAVE_SHM_OPEN)
-    fprintf(stderr, "gr::vmcircbuf_mmap_shm_open: mmap or shm_open is not available\n");
+    std::stringstream error_msg;
+    error_msg << "mmap or shm_open is not available";
+    GR_LOG_ERROR(d_logger, error_msg.str());
     throw std::runtime_error("gr::vmcircbuf_mmap_shm_open");
 #else
     gr::thread::scoped_lock guard(s_vm_mutex);
 
     static int s_seg_counter = 0;
 
     if (size <= 0 || (size % gr::pagesize()) != 0) {
-        fprintf(stderr, "gr::vmcircbuf_mmap_shm_open: invalid size = %d\n", size);
+        std::stringstream error_msg;
+        error_msg << "invalid size =" << size;
+        GR_LOG_ERROR(d_logger, error_msg.str());
```

Comment:
```suggestion
        GR_LOG_ERROR(d_logger, "invalid size =" + std::to_string(size));
```

not going to hold up the PR over this though.